styles.css

```diff
@@ -0,0 +1,446 @@
+    background-color: var(--lightgrey-color);
+    color: var(--black-color);
+    z-index: -2;
+    position: relative;
+}
+
+.case-studies {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 20px 0;
+}
+
+.case-studies-image {
+    font-size: 0; /* to remove white line under picture */
+    flex-basis: 800px;
+    flex-grow: 0;
+    flex-shrink: 1;
+    box-shadow: 0 10px 5px grey;
+}
+
+.case-studies-text {
+    flex-basis: 400px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+}
+
+.squiggle {
+    margin: 20px 0;
+}
+
+.case-studies-1 {
+    flex-direction: row-reverse;
+}
+
+.case-studies-2 {
+
+}
+
+.case-studies-3 {
+    flex-direction: row-reverse;
+}
+
+.view-project-link {
+    text-transform: uppercase;
+    font-weight: bold;
+    text-decoration: none;
+    padding: 50px 0;
+    position: relative;
+}
+
+.view-project-link::after {
+    display: block;
+    content: "";
+    width: 25px;
+    height: 3px;
+    background-color: var(--purple-color);
+    transition: width .35s ease-in-out;
+    position: absolute;
+    left: 150px;
+    bottom: 60px;
+}
+
+.view-project-link:hover::after {
+    width: 50%;
+}
+
+#case-studies-border {
+    padding: 16rem 0 8rem 0;
+}
+
+#case-studies-border::after {
+    content: "";
+    position: absolute;
+    border-top: 0 solid transparent;
+    border-right: 100vw solid white;
+    border-bottom: 70px solid transparent;
+    top: 0;
+    left: 0;
+}
+
+.case-studies-bg {
+    border: 2px solid hotpink;
+    position: relative;
+}
+
+.case-studies-wrapper::before {
+    content: "work";
+    position: absolute;
+    color: white;
+    font-family: 'Roboto', sans-serif;
+    font-weight: 700;
+    font-size: 550px;
+    z-index: -1;
+    top: -20px;
+}
+
+/* --------------- 4.4 Service ---------------------------- */
+.service-wrapper {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    position: relative;
+}
+
+h3 {
+    color: var(--purple-color);
+    padding: 10px 0;
+}
+
+.service {
+    border: 2px solid var(--purple-color);
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    flex-basis: 350px;
+    min-height: 295px;
+    margin: 20px;
+    padding: 30px;
+    color: var(--gray-text-color);
+    font-size: 15px;
+    box-shadow: 0 5px 10px grey;
+}
+
+.service-icon {
+    width: 50px;
+    height: auto;
+}
+
+#service {
+    padding: 16rem 0 8rem 0;
+}
+
+#service::after {
+    content: "";
+    position: absolute;
+    border-top: 0 solid transparent;
+    border-right: 100vw solid var(--lightgrey-color);
+    border-bottom: 70px solid transparent;
+    top: 0;
+    left: 0;
+}
+
+.service-wrapper::after {
+    content: "services";
+    position: absolute;
+    color: var(--bg-words-color);
+    font-family: 'Roboto', sans-serif;
+    font-weight: 700;
+    font-size: 300px;
+    z-index: -1;
```

Comment:
Ik heb deze lokaal nog even geprobeerd uit te zetten, maar visueel zie ik niets veel veranderen. Misschien zie ik het over het hoofd, maar gebruik z-index is eigenlijk een pleister op een ander probleem: namelijk dat de volgorde van je DOM elementen niet goed zit. Misschien had je hier niet de ::after pseudo moeten gebruiken ,maar de ::before

Reply:
heb dit zelf ook nog geprobeerd, er is geen verschil tussen before en after omdat de achtergrond text net boven de blokken staat, als deze iets lager zou komen valt de achtergrond text met z-index er achter, en zonder komt iet er boven op te liggen, ik zla nog eens uitzoeken of ik het kan aanpassen zodat er geen z-index nodig is